### PR TITLE
docs: use placeholder/lipsum shortcode in bug report guide

### PR DESCRIPTION
**Pull request by mcanouil**
This Pull Request update the reproducible examples in the bug report guide to include the lipsum and placeholder shortcode as well as a cross-reference.

### CODE REVIEW

File: bug-reports.qmd [hunk 1]
```diff
@@ -36,14 +36,19 @@ format: html
 engine: jupyter
 ---
 
-This is a minimal self-contained reproducible Quarto document using `format: html` and `jupyter` engine.
+This is a reproducible Quarto document using `format: html`.
 It is written in Markdown and contains embedded Python code.
+When you run the code, it will produce a message.
 
-```{{python}}
+```{python}
```
[the remark this answers is not in the document]
Comment (mcanouil) [reply bug-reports.qmd:43]: No, that's a bad copy/paste from the reprex used in the issue/discussion template.

[the remark this answers is not in the document]
Comment (mcanouil) [reply bug-reports.qmd]: 9e6bcb71e5dd89c88e76bf46d9d78cb14a7a91a8

**Review by cderv — APPROVED** [review]

**Comment by cderv** [comment]
Thanks

**Pull request merged**